Description: Remove unnecessary rust feature flags
Forwarded: not-needed
Author: Antonin Delpeuch <antonin@delpeuch.eu>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -5,14 +5,6 @@
 #![allow(non_camel_case_types)]
 #![allow(non_snake_case)]
 #![allow(non_upper_case_globals)]
-// These are needed for the neon SIMD code and can be removed once the MSRV supports the
-// instrinsics we use
-#![cfg_attr(feature = "neon", feature(stdsimd))]
-#![cfg_attr(
-    feature = "neon",
-    feature(arm_target_feature, raw_ref_op)
-
-)]
 
 /// These values match the Rendering Intent values from the ICC spec
 #[repr(C)]
